### PR TITLE
Wording Tweaks

### PR DESCRIPTION
**Pull request by brianfleming**
## Description

- Favoring Media Clusters instead of just media to reinforce that media is mostly shown in clusters, even if a cluster of 1
- Added subheaders to all tipilne lists for consistency
- Paired claim with fact-check on shared feeds instead of being paired with Media Clusters
- Sort prop types when files are touched
- Random: fix a buttonmain style mistake

## Type of change

- [x] Performance improvement and/or refactoring (non-breaking change that keeps existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Security mitigation or enhancement
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Automated test (add or update automated tests)

## How has this been tested?

manual verification

## Things to pay attention to during code review
This is mostly text additions and changes

## Checklist

- [x] I have performed a self-review of my own code
- [x] I've made sure my branch is runnable and given good testing steps in the PR description
- [x] I considered secure coding practices when writing this code. Any security concerns are noted above.
- [x] I have commented my code in hard-to-understand areas, if any
- [ ] I have made needed changes to the README
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I implemented any new components, they are self-contained, their `propTypes` are declared and they use React Hooks and, if data-fetching is required, they use Relay Modern with fragment containers
- [ ] If my components involve user interaction - specifically button, text fields, or other inputs - I have added a [BEM-like class name](https://meedan.atlassian.net/wiki/spaces/ENG/pages/1327628289/Naming+conventions+for+interactive+elements) to the element that is interacted with
- [x] To the best of my knowledge, any new styles are applied according to the design system
- [ ] If I added a new external dependency, I included a rationale for doing so and an estimate of the change in bundle size (e.g., checked in https://bundlephobia.com/)
- [x] If I touched a file that included an eslint-disable-file header, I updated the code such that the disabler can be removed

### CODE REVIEW

File: src/app/components/cds/buttons-checkboxes-chips/ButtonMain.module.css [hunk 1]
```diff
@@ -311,11 +311,11 @@
     }
 
     &.outlined {
-      @mixin main-button-outlined-theme var(--color-gray-15), var(--color-gray-37), var(--color-gray-15), var(--color-gray-37), transparent;
+      @mixin main-button-outlined-theme var(--color-gray-15), var(--color-gray-37), var(--color-gray-15), var(--color-gray-37), var(--color-gray-96);
```
Comment (brianfleming) [inline src/app/components/cds/buttons-checkboxes-chips/ButtonMain.module.css:314]: Note to reviewers, this is unreleated to goal of this PR, but noticed this button main style issue

**Comment by brianfleming** [comment]
Ok reviewers, will probably continue this process in other PRs, but this is a pretty good stopping point in this level of progress. As described in the description, this PR is mostly re-wording of things to make things more clear or more verbose

**Comment by brianfleming** [comment]
cc @caiosba and @amoedoamorim for visibility

**Review by sarahkeh — APPROVED** [review]
Woo, got through them all... Really nice changes, especially the extra size of the Custom Filtered Lists button.  Looks good to go!

**Pull request merged**